component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -298,7 +298,7 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 	{
 		$strippedmatch = preg_replace('/(#|:|;)+[^}]*/', '', $matchesarray[0][$i]);
 
-		if (in_array($strippedmatch, $search))
+		if (in_array($strippedmatch, $search, FALSE))
```

Comment:
It is, but bought best to include it, as you prefer to call jinput and specify that we are getting with filter CMS for instance but it's the default anyhow.